cloud-sql/postgres/sqlalchemy/main.py

```diff
@@ -36,16 +36,13 @@
 # managing a pool of connections to your database
 db = sqlalchemy.create_engine(
     # Equivalent URL:
-    # postgres+pg8000://<db_user>:<db_pass>@/<db_name>?unix_socket=/cloudsql/<cloud_sql_instance_name>/.s.PGSQL.5432
+    # postgres+psycopg2://<db_user>:<db_pass>@/<db_name>?unix_socket=/cloudsql/<cloud_sql_instance_name>/.s.PGSQL.5432
```

Comment:
Are you sure the parameter is `unix_socket`?  I believe I've seen it as `unix_sock` in some adapters.

Reply:
I'd prefer to [keep the pg8000 adapter and just show the correct parameter](https://github.com/GoogleCloudPlatform/python-docs-samples/pull/2067/files).

Reply:
It's cool if it works with the pg8000 adapter. I only changed it because the pg8000 driver wasn't working as per demo and was raising eroors as mentioned at the top. I tested it with psycopg2 instead and it worked. Because the PR is over a month old, I can't find the exact piece of working code that I tested with but I double checked again and this piece works:

```
db = sqlalchemy.create_engine(
    # Equivalent URL:
    'postgres+psycopg2://{}:{}@/{}?host=/cloudsql/{}'.format(db_user, db_pass, db_name, cloud_sql_instance_name)
    )
```

Also StackOverflow says that ['unix_sock' should be 'host'](https://stackoverflow.com/questions/54967660/connect-to-a-database-over-a-unix-socket-using-sqlalchemy).

Reply:
I'll investigate deeper.

Reply:
And found this in SQLAlchemy's documentation. 
![image](https://user-images.githubusercontent.com/8961079/61752471-6a99b200-adc9-11e9-9d12-565c8a0d5d2e.png)

Moreover it appears that pg8000 does not support connecting over Cloud SQL proxy which is a recommended best practice.

Reply:
I'm testing now, thank you btw for taking the time to make the PR :)

Reply:
I think you are right, just will confirm.

Reply:
The parameters get passed directly to the driver, and thus change depend on which driver you are using. `pg8000` requires `unix_sock`: http://pybrary.net/pg8000/dbapi.html